README.md

```diff
@@ -52,7 +52,10 @@ giving us both centralized and local logs.  (Redundancy is nice).
 		flag.Parse()
 
 		if graylogAddr != "" {
-			gelfWriter, err := gelf.NewWriter(graylogAddr)
```

Comment:
Because the parameters are different, or ought to be, at any rate. A UDP connection has optional compression; TCP connections do not. TLS connections will need more parameters for the key and certificate information.